lib/workers/user_sync_worker.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Notesclub.Workers.UserSyncWorker do
+  @moduledoc """
+  Worker to fetch user info and include it during user creation
+  """
+  alias Notesclub.{Accounts, GithubAPI}
+
+  use Oban.Worker,
+    queue: :github_rest,
+    unique: [period: 300, states: [:available, :scheduled, :executing]]
+
+  @impl Oban.Worker
+  def perform(%Oban.Job{args: %{"user_id" => user_id}}) do
+    with user = %{username: username} <- Accounts.get_user!(user_id),
+         {:ok, user_info} <- GithubAPI.get_user_info(username) do
+      Accounts.update_user(user, user_info)
```

Comment:
Why not this to make sure we update the user?
```suggestion
    with user = %{username: username} <- Accounts.get_user!(user_id),
         {:ok, user_info} <- GithubAPI.get_user_info(username),
         {:ok, _user} = Accounts.update_user(user, user_info) do
```

Reply:
Oh yeah, that's much better! Updated.